### DISCUSSION TITLE
Welcome to start.spring.io Discussions!

### DISCUSSION THREAD

**Discussion by snicoll** [spring-io/start.spring.io#1245] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AeshuAsin** [comment]
Hello New to Java Spring Boot, trying to discover more information about

**Comment by AeshuAsin** [comment]
If any member has any learning path apart from website help. Please help me on this

**Reply by mhalbritter** [reply]
Hello, have you seen the guides at https://spring.io/guides? 

Besides that, we have some videos on Spring, for example Josh Long's channel at https://www.youtube.com/@coffeesoftware or Dan Vega's at https://www.youtube.com/@DanVega.
